server/debian.Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM debian:10-slim
+
+LABEL varion="2.0" \
+    maintainer="ESh4d0w, Markus Hadenfeldt, docker@teaspeak.de, h1dden-da3m0n" \
+    description="A simple TeaSpeak server running on debian 10 (amd64)"
+
+ARG TEASPEAK_VERSION="1.4.12"
+
+RUN	apt-get update && apt-get upgrade -qqy \
+    && apt-get install --no-install-recommends -qqy \
+         ca-certificates wget curl python libnice10 ffmpeg tzdata \
```

Comment:
`libnice10` isn't required anymore.
It now comes in a bundle with TeaSpeak.

Reply:
Sweet, that means that the images can get a lot smaller in size (at least I think that it was libnice10, which bloated them)

Reply:
Yep, libnice has glib2.0 as dependency which is quite a lot.